tests/test_backup.rs

```diff
@@ -26,38 +26,72 @@ use util::DBPath;
 fn backup_restore() {
```

Comment:
I would rather leave the existing test as it is and add a new one, so that we have two tests: one for the `restore_from_latest_backup` method and another one for `restore_from_backup`.

Reply:
oka~